android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -91,7 +91,6 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
     // the valid url at index 0 should be the one that was selected to use for setting up cronet
     private var validUrls = Collections.synchronizedList(mutableListOf<String>())
     private var invalidUrls = Collections.synchronizedList(mutableListOf<String>())
-    private var blockedUrls = Collections.synchronizedList(mutableListOf<String>())
```

Comment:
were you running into issues testing with the blocked list?  in theory debug builds shouldn't skip blocked urls.